src/averageAges.js

```diff
@@ -14,14 +14,18 @@
  *
  * @return {number}
  */
+const averageOfPeople = (people) =>
```

Comment:
The function name should be more descriptive. Consider renaming it to 'calculateAverageAge' instead of 'averageOfPeople'
`[NAMING] - Name of functions should start with verbs. It will be easier to understand a function's purpose.` Checklist p1.